### ISSUE TITLE
Multi-pretrained weights: Add model builders for Classification

### ISSUE THREAD

**Issue by datumbox** [pytorch/vision#4672]
### 🚀 The feature

Update the model builders for all Classification models:
- [x] AlexNet
- [x] DenseNet
- [x] EfficientNet
- [x] GoogleNet
- [x] InceptionV3
- [x] MNASNet
- [x] MobileNetV2
- [x] MobileNetV3
- [x] RegNet
- [x] ResNet
- [x] ShuffleNetV2
- [x] SqueezeNet
- [x] VGG

cc @datumbox @vfdev-5 @pmeier @bjuncek
